web/src/components/dashboard/TwoTabDashboardLayout.tsx

```diff
@@ -0,0 +1,105 @@
+import { getMergedConfig } from "@/contexts/configContext";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { filterCertifications } from "@/lib/domain-logic/filterCertifications";
+import { filterFundings } from "@/lib/domain-logic/filterFundings";
+import { Certification, Funding } from "@/lib/types/types";
+import { getVisibleCertifications, getVisibleFundings, templateEval } from "@/lib/utils/helpers";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
+import { UserData } from "@businessnjgovnavigator/shared/userData";
+import { TabContext, TabList, TabPanel } from "@mui/lab/";
+import Tab from "@mui/material/Tab";
+import * as React from "react";
+import { ReactElement } from "react";
+import { SingleColumnContainer } from "../njwds/SingleColumnContainer";
+
+interface Props {
+  firstTab: ReactElement;
+  secondTab: ReactElement;
+  certifications: Certification[];
+  fundings: Funding[];
+}
+
+export default function TwoTabDashboardLayout({ firstTab, secondTab, certifications, fundings }: Props) {
+  const { userData } = useUserData();
+  const [tabIndex, setTabIndex] = React.useState(0);
+  const Config = getMergedConfig();
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setTabIndex(Number(newValue));
+  };
+
+  const tabStyling = {
+    "& .MuiTabs-indicator": {
+      display: "none",
+    },
+    "& :focus": {
+      outline: "none !important",
+    },
+    "& .Mui-selected": {
+      style: {
+        color: "white",
+      },
+    },
+  };
+  const unselectedButtonStyling = {
+    textTransform: "none",
+    color: "#5c5c5c !important",
+    borderRadius: "8px",
+  };
+  const selectedButtonStyling = {
+    backgroundColor: "white",
+    border: "1px solid #e6e6e6",
+    borderRadius: "8px",
+    textTransform: "none",
+    color: "#1b1b1b !important",
+  };
+
+  const getContentWithCardCount = () => {
+    let count = 0;
+    if (LookupOperatingPhaseById(userData?.profileData.operatingPhase).displayCertifications) {
+      count += getVisibleCertifications(
+        filterCertifications(certifications, userData as UserData),
+        userData as UserData
+      ).length;
+    }
+
+    if (LookupOperatingPhaseById(userData?.profileData.operatingPhase).displayFundings) {
+      count += getVisibleFundings(
+        filterFundings(fundings, userData as UserData),
+        userData as UserData
+      ).length;
+    }
```

Comment:
this is a really clean solution, using the `displayFundings` and `displayCertifications` indicators on the phase as the if-condition. Well done

Reply:
Yay!!!